style-guide.md

```diff
@@ -0,0 +1,111 @@
+	* 	When color schemes convey essential meaning (vital UI elements, etc) they should be functional in grayscale
+* Do not use things that would be a nightmare for screen readers
+ 	* Avoid using non-ASCII symbols unless you are writing something meaningful in another language
+ 		* For example, do not use symbols from U+1D400 𝒔𝒖𝒄𝒉 𝒂𝒔 𝒕𝒉𝒆𝒔𝒆 as a replacement for ***actual bold and italic formatting***
+		* Using actual bold and italics formatting is acceptable, but bear in mind screen readers may not declare something as being bold or italic
+
+## Assumptions about a user's system
+* If talking about a program that does not support certain operating systems, make it clear which systems are supported relatively early in documentation introducing that program
+	* If possible, give an alternative for unsupported OSs
+	* "The Dockstore CLI is only supported on Unix-like operating systems, including Linux and Mac. Windows users may wish to..."
+* Do not assume users are or are not using an HPC
+	* Assume that if they are using an HPC, they at least know how to use their HPC's job submission system (SLURM, etc)
+	* If working with/installing something requiring root permissions such as Docker, mention that root permission is needed once, then assume from that point forward that your HPC users have root privileges
+	* However, do not assume that HPC users know how programs you introduce may work differently on an HPC
+		* Example: Running a WDL on an HPC via Cromwell (e.g. via the Dockstore CLI) requires additional runtime arguments that a local user does not need to include in their WDL program, so make sure to either include those runtime arguments in your examples or tell HPC users what additional things they need
+* Assume Linux users are using Ubuntu or know how to translate into their own distro
+	* Exception: If directing very new users to install software via apt-get, consider mentioning that the instructions are specific to Ubuntu -- UCSC's linux lab for students runs on CentOS (which uses yum instead) so this is not a rare occurrence
+* Keep in mind that some of our users are using Macs, so make a note about these situations in the text if they occur:
+	* If there is an important functional difference between the Linux and the Mac version, such as xargs
+	* If there is an important installation difference between the Linux and the Mac version, such as the installation of the Dockstore CLI
+	* If something does not work/works differently on Apple Silicon Macs
+	* If there is an important difference between the bash and zsh version of something
+		* Do not assume that a Mac user is aware of whether they are using bash or zsh (the default changed in Catalina) or knows how to change between the two
+		* Do not worry about other shells
+
+## Coding in RST
+* Write in RST, not markdown
+* Use `.. comment` to write comments
+* Use consistent indentation spacing within a single RST file
+* Do not break up body text with a newline every x characters
```

Comment:
This is a "do not", but what would be the equivalent "do"? Keeping the body of text on one line?

Reply:
Essentially, yeah. Does https://github.com/dockstore/dockstore-documentation/pull/197/commits/4c347050dae47c30176c6b11c32d97245472b6e7 clear this up?

Reply:
Yeah, looks good